test-lit/semantics/if_arith01.f90

```diff
@@ -1,5 +1,8 @@
 ! Check that a basic arithmetic if compiles.
 
+! RUN: %flang -fdebug-resolve-names -fparse-only %s 2>&1
```

Comment:
So these tests without FileCheck. I imagine they will always pass even if they are not doing the right thing correct? 
I don't understand them quite well, the reason they are this way.

Reply:
These tests just need to not throw any errors when ran.